scrapy/core/downloader/handlers/__init__.py

```diff
@@ -24,6 +24,13 @@ def __init__(self, crawler):
             crawler.settings.getwithbase('DOWNLOAD_HANDLERS'))
         for scheme, clspath in six.iteritems(handlers):
             self._schemes[scheme] = clspath
+        for scheme in self._schemes:
+            path = self._schemes[scheme]
+            dhcls = load_object(path)
+            lazy = getattr(dhcls, 'lazy', False)
```

Comment:
Technically, lazy=False by default is backwards incompatible for custom user download handlers. What do you think about keeping it as True by default, but adding lazy=False for default handlers other than s3?

Reply:
Yeah, I agree